### ISSUE TITLE
Computation of Cohen's Kappa interrater agreement

### ISSUE THREAD

**Issue by bcdasilv** [SentimentAnalysisInSE/issuecomments_analysis#24]
That's another quantitative analysis provided by the benchmark paper we're following.

Cohen's Kappa coefficient provides a quantitative measure of how much agreement there is between two classifications. See the last paragraph of section 4.1 on the benchmark paper (https://arxiv.org/ftp/arxiv/papers/1803/1803.06525.pdf) and also table 4.

Here's a library to do it in Python: https://scikit-learn.org/stable/modules/generated/sklearn.metrics.cohen_kappa_score.html (inside there's also a link to their user guide).

We didn't discuss this issue at our last meeting. If you don't find time to work on it in the next days, that's fine, we'll do it the following week.

**Comment by nauticalmonkey** [comment]
it looks like we just need to read in each of the metrics labels in as arrays and pass it to sklearn.metrics.cohen_kappa_score(y1, y2, labels=None, weights=None, sample_weight=None)
as the y1 and y2. ill give it some work and see what i come out with

**Comment by bcdasilv** [comment]
I think you're right Justin. I used this test years ago but in R (it's a stats language) and the API is similar. That's a pair-wise comparison (limited by the concept of the statistical test itself and not the lib API). If you note table 4 in the paper, they're showing the results in pairs.
